airbyte-integrations/connectors/source-mysql/src/test/java/io/airbyte/integrations/source/mysql/MySqlSSLJdbcSourceAcceptanceTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.mysql;
+
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.string.Strings;
+import io.airbyte.db.Databases;
+import org.jooq.SQLDialect;
+import org.junit.jupiter.api.BeforeEach;
+
+class MySqlSSLJdbcSourceAcceptanceTest extends MySqlJdbcSourceAcceptanceTest {
```

Comment:
Nitpick. `SSL` -> `Ssl`

```suggestion
class MySqlSslJdbcSourceAcceptanceTest extends MySqlJdbcSourceAcceptanceTest {
```

Reply:
Thanks for pointing that out. changed it